bans-core/src/main/java/space/arim/libertybans/core/config/MessagesConfig.java

```diff
@@ -232,6 +238,33 @@ default Component prefix() {
 		})
 		Map<PunishmentType, String> punishmentTypeDisplay();
 
+		@ConfKey("punishment-type-verb-display")
+		@ConfComments("How should punishment types be displayed as a verb? Used for the %TYPE_VERB% variable.")
+		@DefaultMap({
+				"ban", "Banned",
+				"mute", "Muted",
+				"warn", "Warned",
+				"kick", "Kicked"
+		})
+		Map<PunishmentType, String> punishmentTypeVerbDisplay();
+
+		@ConfKey("punishment-expired-display")
+		@ConfComments("How should the %HAS_EXPIRED% variable be displayed?")
+		@SubSection
+		PunishmentExpiredDisplay punishmentActiveDisplay();
```

Comment:
```suggestion
		PunishmentExpiredDisplay punishmentExpiredDisplay();
```
To be consistent with the rest of the code